tools/serverpod_cli/bin/serverpod_cli.dart

```diff
@@ -14,27 +14,13 @@ import 'internal_tools/generate_pubspecs.dart';
 import 'run/runner.dart';
 import 'shared/environment.dart';
 import 'util/command_line_tools.dart';
+import 'util/constants.dart';
 import 'util/print.dart';
 
-const cmdCreate = 'create';
```

Comment:
Breaking out the constants shouldn't be part of this PR. Not sure I think it improves readability of the app either as it is would break up where you need to look for the constants.